src/CmsBundle/Controller/Backend/ConfigController.php

```diff
@@ -17,6 +18,9 @@ class ConfigController extends Controller
      */
     public function indexAction(Request $request)
     {
+        //Check permissions
```

Comment:
Can you remove all these `//Check permissions` comments? The function name `denyAccessUnlessGranted` is already clear enough